ci-operator/config/openshift/managed-upgrade-operator/openshift-managed-upgrade-operator-master.yaml

```diff
@@ -1,53 +1,52 @@
-base_images:
-  base:
-    name: ubi
-    namespace: ocp
-    tag: "8"
-  openshift_release_golang-1.13:
-    name: release
-    namespace: openshift
-    tag: golang-1.13
 build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.13
+  from_repository: true
 images:
 - dockerfile_path: build/Dockerfile
-  from: base
-  to: managed-upgrade-operator
+  to: unused
 resources:
   '*':
+    limits:
+      memory: 10Gi
```

Comment:
Set this back to 4Gi (the standard) once muo has https://github.com/openshift/boilerplate/pull/115.

Reply:
Done via https://github.com/openshift/release/pull/16045